template.rb

```diff
@@ -1,4 +1,8 @@
-gem "dls_template"
+if ENV["GEM_TEST"]
+  gem "dls_template", path: "../."
+else
+  gem "dls_template"
```

Comment:
Why do we want this gem in the generated app?

Reply:
We don't want it, but it needs to be there when `rails generate dls_template:install` is run. That's why there is a final instruction to remove the gem:
https://github.com/pulibrary/dls_template/blob/main/template.rb#L18-L24